src/main/resources/policies/document-rechten.rego

```diff
@@ -77,7 +77,6 @@ verwijderen {
 }
 verwijderen {
     recordmanager.rol in user.rollen
-    document.zaak_open == true
```

Comment:
Do we need to document this?
Do we need to communicate this change?

Reply:
I actually didn't need to change the matrix to align with this, so I guess not, but I'll check with the PO to be sure.